### PR TITLE
Rework data into beacon format

### PR DESCRIPTION
**Pull request by tom-parker**
cc @commuterjoy 

I think this covers everything bar the two outstanding issues on the enrichment repo. I'm wondering whether this one - https://github.com/Financial-Times/spoor-enrichment/issues/79 is too specific to next and should happen here instead?

### CODE REVIEW

**Comment by commuterjoy** [comment]
Looks good. You can ship this by doing a git push to the Heroku app.

**Comment by tom-parker** [comment]
Any objections to me ES6ifying the consumer? Some of this might be slightly cleaner with some destructuring action

**Comment by commuterjoy** [comment]
Please do!

File: sinks/keen.js [hunk 1]
```diff
@@ -17,18 +17,47 @@ var keen = keenIO.configure({
...
 		return;
 	}
-	
+
 	if (message.ingest._bodyFlattened.action !== 'view') {
 		return;
 	}
 
-	if (Math.random() > parseFloat(process.env.keen_pc || 0.1)) {
+	if (message.egest.annotations.url.hostname !== 'next.ft.com') {
```
Comment (commuterjoy) [inline sinks/keen.js]: better do `if (message.egest.annotations.url && message.egest.annotations.url.hostname !== 'next.ft.com') {`

File: sinks/keen.js [hunk 2]
```diff
@@ -17,18 +17,47 @@ var keen = keenIO.configure({
...
 		return;
 	}
 
+	// Translate data to current format for beacon
+	message.egest.annotations.time.weekday = message.egest.annotations.time.isWeekday;
+
+	var page = {
+		location: {
```
Comment (commuterjoy) [inline sinks/keen.js]: can we not just do, 

```
location: message.egest.annotations.url,
referrer: message.egest.annotations.referrer
```

**Comment by commuterjoy** [comment]
+1

**Pull request merged**